packages/yoroi-extension/features/support/windowManager.js

```diff
@@ -0,0 +1,120 @@
+// @flow
+import { WebDriver } from 'selenium-webdriver';
+
+type WindowType = 'tab' | 'window';
+type CustomWindowHandle = {|
+  title: string,
+  handle: string,
+|};
+
+class WindowManagerError extends Error {}
+
+export class WindowManager {
+  windowHandles: Array<CustomWindowHandle>;
+  driver: WebDriver;
+
+  constructor(driver: WebDriver) {
+    this.driver = driver;
+    this.windowHandles = [];
+  }
+
+  async init() {
+    const mainWindowHandle = await this.privateGetCurrentWindowHandle();
+    this.windowHandles.push({ title: 'main', handle: mainWindowHandle });
+  }
+
+  privateGetHandleByTitle(title: string): Array<CustomWindowHandle> {
```

Comment:
use `_foo` convention for "private" functions plz

Reply:
Fixed